src/Libraries/Microsoft.Extensions.AI.OpenAI/OpenAIChatClient.cs

```diff
@@ -362,19 +346,6 @@ private static async IAsyncEnumerable<ChatResponseUpdate> FromOpenAIStreamingCha
                 }
             }
 
-            // Refusals are about the model not following the schema for tool calls. As such, if we have any refusal,
-            // add it to this function calling item.
-            if (refusal is not null)
```

Comment:
If we're no longer using this, we no longer need the local or appending to it?

Reply:
Right, I also removed `fingerprint`.